src/enums/move-category.ts

```diff
@@ -1,5 +1,5 @@
 export enum MoveCategory {
-  PHYSICAL,
-  SPECIAL,
-  STATUS,
+  STATUS = 1,
+  PHYSICAL = 2,
+  SPECIAL = 3,
```

Comment:
these should already be incremented automatically, no need to set 2 and 3
```suggestion
  PHYSICAL,
  SPECIAL,
```